docs/user-guide/installation.rst

```diff
@@ -156,23 +166,14 @@ Server
       accept connections until it finishes. When this happens, Pulp will wait for MongoDB to
       become available before starting.
 
+   .. _configure MongoDB to use SSL: http://docs.mongodb.org/v2.4/tutorial/configure-ssl/#configure-mongod-and-mongos-for-ssl
 
 #. You must also provide a message bus for Pulp to use. Pulp will work with Qpid or RabbitMQ, but
    is tested with Qpid, and uses Qpid by default. This can be on the same host that you will
    run Pulp on, or elsewhere as you please. To install Qpid on a yum based system, use
    this command::
-    
-    $ sudo yum install qpid-cpp-server qpid-cpp-server-store
 
-   .. note::
-      In environments that use Qpid, the ``qpid-cpp-server-store`` package provides durability, a
-      feature that saves broker state if the broker is restarted. This is a required feature for
-      the correct operation of Pulp. Qpid provides a higher performance durability package named
-      ``qpid-cpp-server-linearstore`` which can be used instead of ``qpid-cpp-server-store``, but
-      may not be available on all versions of Qpid. If ``qpid-cpp-server-linearstore`` is available
-      in your environment, consider uninstalling ``qpid-cpp-server-store`` and installing
-      ``qpid-cpp-server-linearstore`` instead for improved broker performance. After installing
-      this package, you will need to restart the Qpid broker to enable the durability feature.
+    $ sudo yum install qpid-cpp-server qpid-cpp-server-linearstore qpid-tools
```

Comment:
On Mon, Dec 07, 2015 at 02:21:16PM -0800, bmbouter wrote:
> @rbarlow For now, we can assume that all installations are using comps.xml currently, since it's the recommended method of installing qpid client bits in the installation guide:  https://github.com/pulp/pulp/blame/master/docs/user-guide/installation.rst#L199
> 
> I can't think of any advantages to not using the comps.xml. What are some?

Not all automation supports installation with groups. For example, last
I checked the Ansible dnf module did not support groups. I haven't tried
in in ~6 months though.

In addition, I personally don't tend to install Pulp with groups when I
make test installations. I prefer to pick the packages I want to use by
hand. I suspect that I'm not the only user with that preference.

Therefore I do not think that we can assume that *all* installations use
comps.xml.

-- 
Randy Barlow
irc:   bowlofeggs
phone: +1-919-890-8150